codebase/config/sync/migrate_plus.migration.idc_ingest_media_document.yml

```diff
@@ -77,13 +77,14 @@ process:
   field_media_document/target_id:
     plugin: file_import
     source: '@_download_filepath'
+    uid: constants/ADMIN
```

Comment:
Big Picture: Files are "owned by" the Admin user.

Reply:
Yes, apparently unowned files (or anonymous) violate Drupal's assumption of validity